### PR TITLE
[WIP] Colab-user-guide

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by jason-dai** [comment]
Do you have the test page?

**Comment by Le-Zheng** [comment]
> Do you have the test page?

test page: https://le-colabdoc.readthedocs.io/en/latest/doc/UserGuide/colab.html

File: readthedocs/source/doc/UserGuide/colab.md [hunk 1]
```diff
@@ -2,20 +2,60 @@
 
 ---
 
-You can use Analytics Zoo without any installation by using  [Google Colab](https://colab.research.google.com/). Analytics Zoo also include a collection of [notebooks]() that can be directly opened and run in Colab.
+You can use Analytics Zoo without any installation by using  [Google Colab](https://colab.research.google.com/). Analytics Zoo also include a collection of [notebooks](./notebooks.md) that can be directly opened and run in Colab.
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/colab.md:5]: Move "Analytics Zoo also include a collection of [notebooks](./notebooks.md) that can be directly opened and run in Colab." to the next section.

File: readthedocs/source/doc/UserGuide/colab.md [hunk 2]
```diff
@@ -2,20 +2,60 @@
...
 
 ### **1. Open a Colab Notebook**
 
-<TODO: add instructions for opening notebooks in `docs/colab-notebook/`>
+Find the [notebooks](./notebooks.md) running on Colab.
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/colab.md:9]: Need to talk about the "Run Anyway" button.  See https://course.fast.ai/start_colab

File: readthedocs/source/doc/UserGuide/colab.md [hunk 3]
```diff
@@ -2,20 +2,60 @@
...
+!apt-get install openjdk-8-jdk-headless -qq > /dev/null
+# Set jdk environment path which enables you to run Pyspark in your Colab environment.
+import os
+os.environ["JAVA_HOME"] = "/usr/lib/jvm/java-8-openjdk-amd64"
+!update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+```
+
 **Install Conda**
 
+If the application requires Conda, you can execute the code bellow on Colab to install Conda.
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/colab.md:30]: Run the code blow to install [conda](https://docs.conda.io/) on Colab.

File: readthedocs/source/doc/UserGuide/colab.md [hunk 4]
```diff
@@ -2,20 +2,60 @@
...
+import sys
+_ = (sys.path
+        .append("/usr/local/lib/python3.6/site-packages"))
+
+os.environ['PYTHONHOME']="/usr/local"
+```
+
 **Install Analytics Zoo**
 
+You can install the latest pre-release version using `pip install --pre analytics-zoo`.
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/colab.md:52]: Install the latest pre-release version using `pip install --pre analytics-zoo`.

File: readthedocs/source/doc/UserGuide/colab.md [hunk 5]
```diff
@@ -2,11 +2,11 @@
 
 ---
 
-You can use Analytics Zoo without any installation by using  [Google Colab](https://colab.research.google.com/). Analytics Zoo also include a collection of [notebooks](./notebooks.md) that can be directly opened and run in Colab.
+You can use Analytics Zoo without any installation by using  [Google Colab](https://colab.research.google.com/).
 
 ### **1. Open a Colab Notebook**
 
-Find the [notebooks](./notebooks.md) running on Colab.
+Analytics Zoo includes a collection of [notebooks](./notebooks.md) that can be directly opened and run in Colab. When you open the notebook, you can click ‘Runtime’ tab > 'Run all' to execute all notebook cells. Or click the "run" triangle on the left of each cell to run one by one. When you run the first cell, you may face a pop-up saying 'Warning: This notebook was not authored by Google'; you should click on 'Run Anyway' to get rid of the warning.
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/colab.md:9]: I don't think `Run all` works

File: readthedocs/source/doc/UserGuide/colab.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ os.environ["JAVA_HOME"] = "/usr/lib/jvm/java-8-openjdk-amd64"
 
 **Install Conda**
 
-If the application requires Conda, you can execute the code bellow on Colab to install Conda.
+Run the code bellow to install [Conda](https://docs.conda.io/en/latest/) on Colab.
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/colab.md:30]: conda

**Comment by jason-dai** [comment]
LGTM

File: readthedocs/source/doc/UserGuide/notebooks.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+- **Keras 2.3 Quickstart**
+
+   <a target="_blank" href="https://colab.research.google.com/github/intel-analytics/analytics-zoo/blob/master/docs/docs/colab-notebook/orca/quickstart/keras_lenet_mnist.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>&nbsp; <a target="_blank" href="https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/colab-notebook/orca/quickstart/keras_lenet_mnist.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View source on GitHub</a>
+
+- **PyTorch Quickstart**
+
+    <a target="_blank" href="https://colab.research.google.com/github/intel-analytics/analytics-zoo/blob/master/docs/docs/colab-notebook/orca/quickstart/pytorch_lenet_mnist.ipynb"><img src="https://www.tensorflow.org/images/colab_logo_32px.png" />Run in Google Colab</a>&nbsp; <a target="_blank" href="https://github.com/intel-analytics/analytics-zoo/blob/master/docs/docs/colab-notebook/orca/quickstart/pytorch_lenet_mnist.ipynb"><img src="https://www.tensorflow.org/images/GitHub-Mark-32px.png" />View source on GitHub</a>
+
```
Comment (jason-dai) [inline readthedocs/source/doc/UserGuide/notebooks.md:18]: Add TF2 quick start

Comment (Le-Zheng) [reply readthedocs/source/doc/UserGuide/notebooks.md:22]: done

**Pull request merged**